builder/Makefile.release

```diff
@@ -157,7 +157,7 @@ release: \
 	$(MAKE) -f builder/Makefile.tracee-make ubuntu-prepare
 	BTFHUB=1 STATIC=1 $(MAKE) -f builder/Makefile.tracee-make ubuntu-make ARG="all"
 	# create the tar ball and checksum files
-	$(CMD_TAR) --exclude="*.so" -czf $(OUT_ARCHIVE) $(RELEASE_FILES)
```

Comment:
@rafaeldtinoco maybe you remember - did we have a reason to exclude the .so in first place?

Reply:
Yes, we did not include the golang signatures, only the rego ones, before. After we moved from rego to golang signatures by default, we never changed the Makefile.release to reflect that!